server/data/entryData.js

```diff
@@ -0,0 +1,74 @@
+const entryData = [
```

Comment:
Are you using this entry data anywhere? is this being added to the DB?

Reply:
We are using entryData to seed our database! We are inserting entryData to our local 'entries' database in our entries.seeds.js file.